src/error/syntaxError.js

```diff
@@ -21,15 +44,29 @@ export function syntaxError(
   position: number,
   description: string
 ): GraphQLError {
-  const location = getLocation(source, position);
-  const error = new GraphQLError(
-    `Syntax Error ${source.name} (${location.line}:${location.column}) ` +
-    description + '\n\n' + highlightSourceAtLocation(source, location),
-    undefined,
+  return formatError(
     source,
-    [ position ]
+    position,
+    `Syntax Error ${source.name}`,
+    description,
   );
-  return error;
+}
+
+/**
+ * Produces a GraphQLError for the invariant(...) function that renders the
+ * location where a validation error occurred. If no source is passed in, it
+ * returns an error containing the message, without context.
+ */
+export function validationError(
+  message: string,
```

Comment:
nit, but should we order the argument similar to the functions above? That probably means to move `source` to the top.